File: src/main/java/flustix/fluxifyed/console/commands/StopConsoleCommand.java
package flustix.fluxifyed.console.commands;

import flustix.fluxifyed.console.ConsoleCommand;
import flustix.fluxifyed.console.ConsoleCommands;

public class StopConsoleCommand extends ConsoleCommand {
    public StopConsoleCommand() {
        super("stop", "Stops the bot.");
    }

    @Override
    public void execute(String[] args) {
        ConsoleCommands.LOGGER.info("Stopping bot...");
        System.exit(0);
    }
}

File: Fluxifyed/CC/Commands/StopConsoleCommand.cs
namespace Fluxifyed.CC.Commands; 

public class StopConsoleCommand : IConsoleCommand {
    public string Name => "stop";
    public string Usage => "stop";
    public string Description => "Stops the bot";
    
    public void Execute(string[] args) {
        Console.Write("Stopping bot...");
        ConsoleCommands.Stop();
        Environment.Exit(0);
    }
}